diesel_migrations/migrations_internals/src/lib.rs

```diff
@@ -216,7 +216,7 @@ pub fn revert_migration_with_version<Conn: Connection>(
 ) -> Result<(), RunMigrationsError> {
     migration_with_version(migrations_dir, ver)
         .map_err(|e| e.into())
-        .and_then(|m| revert_migration(conn, m, output))
+        .and_then(|m| revert_migration(conn, &m, output))
```

Comment:
Why is this here ?

Reply:
Because Rust is super inconsistent about whether it runs lints or not, and suddenly this clippy lint (which should have always been failing) is now failing as a result of these unrelated changes.

Reply:
Lol, okay'